src/raylib/core/Audio.hpp

```diff
@@ -0,0 +1,45 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Audio
+*/
+
+#ifndef RAYLIB_CORE_AUDIO_HPP_
+#define RAYLIB_CORE_AUDIO_HPP_
+
+extern "C"
+{
+#include <raylib.h>
+}
+
+/// RayLib namespace
+namespace raylib
+{
+    /// Core namespace inside of raylib
+    namespace core
+    {
+        /// The Audio static class that will initialize audio and manage it
+        class Audio {
+          public:
+            /// Instant audio device
+            static void initAudioDevice();
```

Comment:
The Class is Audio, you don't have need to say "initAudioDevice" you can use initDevice. We know that function is use on Audio Object.